src/vm/wren_vm.h

```diff
@@ -125,6 +125,10 @@ struct WrenVM
 //   [oldSize] will be zero. It should return NULL.
 void* wrenReallocate(WrenVM* vm, void* memory, size_t oldSize, size_t newSize);
 
+// Prints an error to the configuration's `errorFn`, if non-NULL.
+// Otherwise, prints to stderr.
+void wrenPrintError(WrenVM* vm, const char* format, ...);
```

Comment:
How about we move this function to wren_utils.h/.c? wren_vm.c is getting a little cluttered, and this function is fairly separate from the rest of the VM.